source/guides/administration.rst

```diff
@@ -34,197 +34,131 @@ The basics
-    :hidden:
-
-    Shared channels </onboard/shared-channels>
-    Store Mattermost configuration in the database </configure/configuation-in-mattermost-database>
-    Email templates </configure/email-templates>
-    Configuring CloudFront to host Mattermost static assets </configure/configuring-cloudfront-to-host-mattermost-static-assets>
-    Use an outbound proxy </configure/using-outbound-proxy>
-    Chinese, Japanese, and Korean search </configure/enabling-chinese-japanese-korean-search>
-
-* :doc:`Shared channels </onboard/shared-channels>` - Connect channels from multiple Mattermost servers in a federated architecture.
-* :doc:`Include configuration in the Mattermost database </configure/configuation-in-mattermost-database>` - Store Mattermost configuration information in your database rather than as a JSON file. We recommend this for High Availability environments.
-* :doc:`Email templates </configure/email-templates>` - Alter the transactional emails Mattermost sends to your users.
-* :doc:`Configuring CloudFront to host Mattermost static assets </configure/configuring-cloudfront-to-host-mattermost-static-assets>` - Improve caching performance to reduce content load times.
-* :doc:`Using an outbound proxy </configure/using-outbound-proxy>` - Monitor outbound traffic and control the websites that can appear in embedded content.
-* :doc:`Chinese, Japanese, and Korean search </configure/enabling-chinese-japanese-korean-search>` - Set up search capabilities for teams communicating via Chinese, Japanese, or Korean.
-
-User onboarding
-^^^^^^^^^^^^^^^
-
-.. toctree::
-    :maxdepth: 1
-    :hidden:
-
-    Provisioning Workflows </onboard/user-provisioning-workflows>
-    Bulk loading data </onboard/bulk-loading-data>
-    Migration guide </onboard/migrating-to-mattermost>
-
-* :doc:`Provisioning workflows </onboard/user-provisioning-workflows>` - Learn how to provision and de-provision user accounts.
-* :doc:`Bulk loading data </onboard/bulk-loading-data>` - Import bulk data into Mattermost for teams, channels, users, post content, and more.
-* :doc:`Migration guide </onboard/migrating-to-mattermost>` - Learn how to migrate from other chat services to Mattermost.
-
-Advanced user management configurations
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Advanced user management
+~~~~~~~~~~~~~~~~~~~~~~~~
 
 .. toctree::
     :maxdepth: 1
     :hidden:
 
     AD/LDAP groups </onboard/ad-ldap-groups-synchronization>
+    Use AD/LDAP synchronized groups to manage team or private channel membership </onboard/managing-team-channel-membership-using-ad-ldap-sync-groups>
     Guest accounts </onboard/guest-accounts>
-    Using AD/LDAP synchronized groups to manage team or private channel membership </onboard/managing-team-channel-membership-using-ad-ldap-sync-groups>
     SAML Single Sign-On </onboard/sso-saml>
     SAML Single-Sign-On: technical documentation </onboard/sso-saml-technical>
     SSL client certificate setup </onboard/ssl-client-certificate>
```

Comment:
Are SSL client certificate and certificate-based auth available in Cloud? I'm not sure

Reply:
Confirmed with Maria that these two self-hosted features are not supported via Cloud instances.